firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/AppData.java

```diff
@@ -61,6 +62,14 @@ public static AppData create(
         developmentPlatformProvider);
   }
 
+  private static String getAppBuildVersion(PackageInfo packageInfo) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+      return Long.toString(packageInfo.getLongVersionCode());
+    } else {
+      return Integer.toString(packageInfo.versionCode);
```

Comment:
Should we suppress the deprecation message? 

`@Suppress("DEPRECATION") packageInfo.versionCode`

Reply:
👍